app/views/shared/_donation_platforms.html.haml

```diff
@@ -0,0 +1,25 @@
+.row
+  .columns.small-12
+    %h2 How to support us
```

Comment:
Does this h2 header need to be i18n too like the rest of this file?

Reply:
@KimberleyCook sure, can add it to the locales